src/components/Rockets.js

```diff
@@ -25,6 +25,7 @@ const Rockets = () => {
 
             <p className={style.rocketDec}>{racket.description}</p>
             <button type="button" className={style.reserve} onClick={() => dispatch(bookRockets(racket.id))}>Reserve rocket</button>
+            <button type="button" className={style.cancel} onClick={() => dispatch(cancleRockets(racket.id))}>Cancle Reserved rocket</button>
```

Comment:
[OPTIONAL] Consider fixing the typo `cancle`. Rather right `Cancel Reservation` as per the project requirement.